bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/item/ItemResource.java

```diff
@@ -517,6 +518,24 @@ public Response createOrUpdateItem(
     }
 
     /**
+     * helper: Create new item with name and type
+     *
+     * @param itemType type of the item
+     * @param itemname name of the item
+     * @return
```

Comment:
I like that you add JavaDoc also for a private method.
Bbut perhaps you can also add something to the return parameter instead of leaving it empty e.g. <code>@return the newly created item</code>